portal/src/app/app.routes.ts

```diff
@@ -0,0 +1,3 @@
+import { Routes } from '@angular/router';
```

Comment:
I am not sure about this usecase of angular 17 as I have not yet tried. Assuming this is correct way for now.

Reply:
this was beacause of standalone, this is now fixed, we are using routing module and app module just like osmox